static/app/views/alerts/rules/issue/setupMessagingIntegrationButton.tsx

```diff
@@ -22,9 +22,16 @@ interface ProjectWithAlertIntegrationInfo extends Project {
 type Props = {
   projectSlug: string;
   refetchConfigs: () => void;
+  analyticsParams?: {
+    view: string;
```

Comment:
nit: better to make this an enum rather than a string, so you can say "this will always be a string with this value".